library.properties

```diff
@@ -0,0 +1,9 @@
+name=RF24Log
+version=1.0.0
+author=nRF24
+maintainer=nRF24
+sentence=RF24Log
+paragraph=RF24Log
+category=Log
+url=https://github.com/nRF24/RF24Log
+architectures=avr

```

Comment:
Is there a reason it's specific to AVR?

Reply:
I just tested this on Arduino Nano, so I put *avr*. This library.properties needs to be improved - I just did a very minimalistic version so Arduino IDE will not complain.

Reply:
It can probably be a wildcard there, there does not seem to be anything that isn't handled by the Arduino HAL.

Reply:
The Arduino CLI workflow I'm working on only tests against any AVR board that supports SPI (Gemma is the only one excluded). I remember in one of the various issue comments, you said something about a whitelist of supported platforms. Is there something more explicit than `*`?

Reply:
Yes, a comma separated list. It'll just throw a warning if it isn't on the list of supported platforms. I don't think there's a need to start constructing this list because everything Arduino would work.

Reply:
Done by @2bndy5, we have a ```*``` now.